R/RtGam.R

```diff
@@ -26,6 +93,95 @@ RtGam <- function(cases,
   validate(cases, reference_date, group)
 
   df <- prepare_inputs(cases, reference_date, group)
+  formula <- formula_creator(
+    n_timesteps = length(unique(df[["timesteps"]])),
+    k = k,
+    is_grouped = !rlang::is_null(group)
+  )
 
   invisible(NULL)
 }
+
+#' Propose total basis dimensionality from number of data points
+#'
+#' Guess a reasonable value for the `k` argument of [RtGam] based on the number
+#' of data points. This guess **may not work** and almost certainly is not the
```

Comment:
"Return" rather than "Guess"

Reply:
What does it mean to "not work" -- would rephrase as "may be a poor choice"

Reply:
Rewritten!